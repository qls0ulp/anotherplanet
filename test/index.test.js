/* eslint-env jest */

import Enzyme from 'enzyme'
import React from 'react'
import renderer from 'react-test-renderer'
import App from '../pages/index.js'
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({ adapter: new Adapter() });

describe('With Enzyme', () => {
  it('App shows "Another Planet.io"', () => {
    const app = Enzyme.shallow(<App />)
    expect(app.find('h1').text()).toEqual("Another Planet.io")
  })
})

describe('With Snapshot Testing', () => {
  it('App shows "Another Planet.io"', () => {
    const component = renderer.create(<App />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
