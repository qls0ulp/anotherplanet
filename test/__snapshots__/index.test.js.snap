// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With Snapshot Testing App shows "Another Planet.io" 1`] = `
<div
  data-jsx={571573584}
>
  <header
    data-jsx={571573584}
  />
  <div>
    <main
      className="homepage"
      data-jsx={3457201880}
    >
      <section
        className="hero"
        data-jsx={3457201880}
      >
        <div
          className="heroCanvasContainer"
          data-jsx={2747020459}
        />
        <div
          className="scroll-view "
          data-jsx={3457201880}
        >
          <div
            className="main-name"
            data-jsx={3457201880}
          >
            <h1
              data-jsx={3457201880}
            >
              Another Planet.io
            </h1>
          </div>
          <div
            className="baseline"
            data-jsx={3457201880}
          >
            <p
              data-jsx={3457201880}
            >
              Thierry Charbonnel 
              <span
                className="main__xs-inline-none___1qzZQ"
                data-jsx={3457201880}
              >
                –
              </span>
              <span
                className="main__xs-inline___2TdTk"
                data-jsx={3457201880}
              >
                <br
                  data-jsx={3457201880}
                />
              </span>
               UX / UI and Code Designer
              <br
                data-jsx={3457201880}
              />
              <span
                data-jsx={3457201880}
                style={
                  Object {
                    "fontSize": "0.9em",
                    "opacity": 0.5,
                  }
                }
              >
                NYC + Paris
              </span>
            </p>
          </div>
        </div>
        <div
          className="scroll-icon relase"
          data-jsx={3457201880}
        >
          <svg
            data-jsx={3457201880}
            height="48px"
            version="1.1"
            viewBox="0 0 48 48"
            width="48px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              data-jsx={3457201880}
              fill="none"
              fillRule="evenodd"
              id="Page-1"
              stroke="none"
            >
              <g
                data-jsx={3457201880}
                id="scroll-icon"
              >
                <polyline
                  className="arrowBottom"
                  data-jsx={3457201880}
                  points="38.5566864 25.8789063 24.4145508 40.0210419 10.2724152 25.8789062"
                  stroke="#FFFFFF"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                />
                <polyline
                  className="arrowMiddle"
                  data-jsx={3457201880}
                  points="38.5566864 17.8789063 24.4145508 32.0210419 10.2724152 17.8789062"
                  stroke="#FFFFFF"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                />
                <polyline
                  className="arrowUp"
                  data-jsx={3457201880}
                  points="38.5566864 9.87890625 24.4145508 24.0210419 10.2724152 9.87890625"
                  stroke="#FFFFFF"
                  strokeLinecap="round"
                  strokeLinejoin="round"
                  strokeWidth="2"
                />
              </g>
            </g>
          </svg>
        </div>
      </section>
      <div
        className="main__mdc-layout-grid___gQcgz ux"
      >
        <div
          className="main__mdc-layout-grid__inner___1ZRyv"
        >
          <div
            className="main__mdc-layout-grid__cell___fCP2v main__mdc-layout-grid__cell--span-8-tablet___TJQhx main__mdc-layout-grid__cell--span-12-desktop___3LWuk main__mdc-layout-grid__cell--align-middle___2IP35"
          >
            <div
              className="intro"
              data-jsx={3457201880}
            >
              <p
                data-jsx={3457201880}
              >
                I am Thierry Charbonnel, a Designer based in NYC. Specializing in 
                <b
                  data-jsx={3457201880}
                >
                  front-end web development
                </b>
                , prototyping and 
                <b
                  data-jsx={3457201880}
                >
                  UX/UI design
                </b>
                .
              </p>
              <p
                data-jsx={3457201880}
              >
                My focus has been on User Experience (UX), responsive design, componentized systems (atomic), graphic design, motion design and Data Visualization.
              </p>
              <p
                data-jsx={3457201880}
              >
                I am thinking about the intersection of technology + design.
              </p>
              <p
                data-jsx={3457201880}
              >
                My specialty is crafting user experiences which help businesses achieve their goals. I am passionate thinkers and makers, and I love what I do.
              </p>
            </div>
          </div>
        </div>
      </div>
      <div
        className="main__mdc-layout-grid___gQcgz code"
      >
        <div
          className="main__mdc-layout-grid__inner___1ZRyv"
        >
          <div
            className="main__mdc-layout-grid__cell___fCP2v main__mdc-layout-grid__cell--span-8-tablet___TJQhx main__mdc-layout-grid__cell--span-12-desktop___3LWuk"
          >
            <svg
              height="80"
              viewBox="0 0 80 80"
              width="80"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
                fillRule="evenodd"
                stroke="#FFF"
                strokeLinecap="round"
                strokeLinejoin="round"
                strokeWidth="2"
              >
                <path
                  d="M38.937 35.7l-10.325 7.229a1 1 0 0 1-1.393-.246l-.574-.819-6.553 4.589.574.819a1 1 0 0 1-.246 1.393l-4.915 3.441a1 1 0 0 1-1.393-.246L8.377 43.67a1 1 0 0 1 .245-1.393l4.915-3.441a1 1 0 0 1 1.393.245l.574.82 6.553-4.59-.574-.818a1 1 0 0 1 .246-1.393l10.649-7.457-.574-.819a1 1 0 0 1 .246-1.392l10.649-7.457-1.147-1.638a1 1 0 0 1 .245-1.393l9.83-6.883a1 1 0 0 1 1.393.246L64.49 22.689a1 1 0 0 1-.245 1.392l-9.83 6.883a1 1 0 0 1-1.393-.245l-1.147-1.638-7.502 5.252a3 3 0 1 1-5.437 1.366zM42 39v28.537M42 39l16 35.561M42 39L26 74.561M65.376 17.965c-.171-4.892-3.247-8.997-7.511-10.716"
                />
              </g>
            </svg>
            <p
              data-jsx={3457201880}
            >
              Beyond the scene
            </p>
            <p
              className="github"
              data-jsx={3457201880}
            >
              <a
                data-jsx={3457201880}
                href="https://github.com/thierryc/anotherplanet?ref=anotherplanet"
                rel="noopener"
                target="_blank"
              >
                Browse this React, Three.js and Next.js website code on Github
              </a>
            </p>
          </div>
        </div>
        <div
          className="main__mdc-layout-grid__inner___1ZRyv about-icons"
        >
          <div
            className="main__mdc-layout-grid__cell___fCP2v main__mdc-layout-grid__cell--span-2-phone___1V5H6 main__mdc-layout-grid__cell--span-2-tablet___3j6ZO main__mdc-layout-grid__cell--span-3-desktop___bAsKB"
          >
            <svg
              height="128"
              viewBox="0 0 128 128"
              width="128"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="none"
                fillRule="evenodd"
                transform="translate(23 28)"
              >
                <circle
                  cx="41"
                  cy="36"
                  fill="#FFF"
                  fillRule="nonzero"
                  r="7"
                />
                <path
                  d="M40.93 20.644c10.103 0 19.489 1.45 26.565 3.886 8.527 2.936 13.77 7.385 13.77 11.414 0 4.199-5.557 8.926-14.713 11.96-6.923 2.293-16.032 3.49-25.623 3.49-9.833 0-19.144-1.124-26.143-3.516C5.929 44.851.594 40.063.594 35.944c0-3.996 5.006-8.411 13.412-11.342 7.104-2.477 16.721-3.958 26.923-3.958z"
                  stroke="#FFF"
                  strokeWidth="3"
                />
                <path
                  d="M27.61 28.338c5.048-8.752 10.993-16.158 16.638-21.072 6.803-5.92 13.276-8.238 16.767-6.225 3.637 2.097 4.956 9.272 3.01 18.719-1.472 7.143-4.985 15.632-9.777 23.94-4.912 8.518-10.537 16.023-16.106 20.89-7.047 6.161-13.86 8.39-17.428 6.333-3.462-1.997-4.786-8.538-3.125-17.285 1.403-7.39 4.925-16.462 10.021-25.3z"
                  stroke="#FFF"
                  strokeWidth="3"
                />
                <path
                  d="M27.623 43.722c-5.062-8.744-8.51-17.593-9.947-24.938-1.731-8.85-.507-15.616 2.98-17.635 3.634-2.103 10.508.337 17.721 6.741 5.453 4.842 11.053 12.126 15.858 20.426 4.927 8.51 8.62 17.131 10.056 24.387 1.817 9.182.346 16.198-3.218 18.261-3.459 2.003-9.788-.117-16.536-5.924-5.702-4.907-11.802-12.488-16.914-21.318z"
                  stroke="#FFF"
                  strokeWidth="3"
                />
              </g>
            </svg>
          </div>
          <div
            className="main__mdc-layout-grid__cell___fCP2v main__mdc-layout-grid__cell--span-2-phone___1V5H6 main__mdc-layout-grid__cell--span-2-tablet___3j6ZO main__mdc-layout-grid__cell--span-3-desktop___bAsKB"
          >
            <svg
              height="128"
              viewBox="0 0 128 128"
              width="128"
              xmlns="http://www.w3.org/2000/svg"
            >
              <g
                fill="#FFF"
                fillRule="nonzero"
              >
                <path
                  d="M35.125 46.768H57.71v1.788H37.195v13.458h19.292v1.788H37.195v14.775h20.75v1.788h-22.82V46.768zm24.608 0h2.4l10.634 14.775 10.868-14.775L98.42 28 74.131 63.096l12.516 17.27h-2.494L72.767 64.648 61.333 80.365h-2.447l12.61-17.269-11.763-16.328zm27.808 1.788v-1.788h25.737v1.788h-11.857v31.81h-2.07v-31.81H87.54zM7 46.701h2.588l35.685 53.191-14.747-19.594L9.164 49.242 9.07 80.298H7zM112.74 77.586c.428 0 .74-.326.74-.748a.727.727 0 0 0-.74-.748.73.73 0 0 0-.74.748.73.73 0 0 0 .74.748zm2.035-1.968c0 1.245.9 2.059 2.215 2.059 1.4 0 2.246-.839 2.246-2.296v-5.127h-1.128v5.122c0 .81-.408 1.241-1.128 1.241-.645 0-1.083-.402-1.098-1h-1.107zm5.936-.066c.08 1.296 1.163 2.125 2.775 2.125 1.722 0 2.8-.87 2.8-2.255 0-1.09-.615-1.693-2.105-2.04l-.801-.195c-.947-.221-1.33-.518-1.33-1.035 0-.653.595-1.08 1.486-1.08.846 0 1.43.417 1.536 1.085h1.098c-.066-1.22-1.143-2.08-2.619-2.08-1.586 0-2.643.86-2.643 2.15 0 1.065.599 1.698 1.913 2.004l.937.226c.961.226 1.384.558 1.384 1.11 0 .643-.664 1.11-1.57 1.11-.972 0-1.647-.437-1.743-1.125h-1.118z"
                />
              </g>
            </svg>
          </div>
          <div
            className="main__mdc-layout-grid__cell___fCP2v main__mdc-layout-grid__cell--span-2-phone___1V5H6 main__mdc-layout-grid__cell--span-2-tablet___3j6ZO main__mdc-layout-grid__cell--span-3-desktop___bAsKB"
          >
            <svg
              height="128"
              viewBox="0 0 128 128"
              width="128"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M28 28h72v72H28V28zm48.38 56.263l-5.499 3.194c1.986 3.925 6.046 6.919 12.33 6.919 6.426 0 11.21-3.337 11.21-9.429 0-5.65-3.245-8.163-8.994-10.628l-1.691-.725c-2.903-1.257-4.16-2.079-4.16-4.108 0-1.643 1.256-2.9 3.238-2.9 1.943 0 3.194.82 4.354 2.9l5.269-3.383c-2.229-3.92-5.322-5.417-9.623-5.417-6.043 0-9.909 3.863-9.909 8.937 0 5.508 3.244 8.114 8.126 10.194l1.691.726c3.086 1.35 4.926 2.171 4.926 4.491 0 1.936-1.79 3.337-4.594 3.337-3.337 0-5.224-1.74-6.675-4.108zm-23.901.586l-5.508 3.334c1.596 3.377 4.733 6.18 10.15 6.18 5.996 0 10.103-3.188 10.103-10.194V61.072h-6.766v23.006c0 3.381-1.402 4.251-3.625 4.251-2.324 0-3.291-1.594-4.354-3.48z"
                fill="#FFF"
                fillRule="nonzero"
              />
            </svg>
          </div>
          <div
            className="main__mdc-layout-grid__cell___fCP2v main__mdc-layout-grid__cell--span-2-phone___1V5H6 main__mdc-layout-grid__cell--span-2-tablet___3j6ZO main__mdc-layout-grid__cell--span-3-desktop___bAsKB"
          >
            <svg
              height="128"
              viewBox="0 0 128 128"
              width="128"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M9 52.27h13.443v1.953h-5.726v17.385h-2.21V54.223H9V52.27zm16.611 19.338V51h2.583c0 .02.005.031.016.031.01 0 .015.01.015.031 0 .062-.047.15-.14.263-.093.114-.16.253-.202.419v8.212a6.592 6.592 0 0 1 2.1-2.076c.84-.517 1.696-.775 2.567-.775.644 0 1.23.108 1.759.325.529.217.99.563 1.384 1.038.394.476.695 1.085.903 1.829.207.744.31 1.653.31 2.727v8.584h-2.208v-8.522c0-1.488-.255-2.531-.763-3.13-.508-.6-1.125-.899-1.851-.899-.477 0-.955.104-1.432.31a5.71 5.71 0 0 0-1.384.868 3.713 3.713 0 0 0-1.027 1.363c-.239.537-.358 1.157-.358 1.86v8.15H25.61zm16.3-14.162h2.334l-.062 2.727a4.712 4.712 0 0 1 2.07-2.278 6.127 6.127 0 0 1 3.033-.79c1.535 0 2.8.516 3.797 1.55l-1.027 2.138c-.229-.31-.436-.569-.623-.775a3.215 3.215 0 0 0-.606-.527 2.852 2.852 0 0 0-.716-.34 3.155 3.155 0 0 0-.95-.125c-.684 0-1.327.16-1.928.48a5.315 5.315 0 0 0-1.572 1.271 6.456 6.456 0 0 0-1.073 1.813 5.76 5.76 0 0 0-.405 2.138v6.91h-2.272V57.447zm19.304-.341c.747 0 1.473.134 2.178.403a5.04 5.04 0 0 1 1.852 1.224c.529.547.954 1.245 1.275 2.092.322.847.483 1.849.483 3.006a8.676 8.676 0 0 1-.062 1.022H56.827c.042.992.213 1.823.514 2.495.3.671.68 1.214 1.136 1.627.456.413.964.707 1.524.883a5.59 5.59 0 0 0 1.68.263c1.474 0 2.698-.526 3.673-1.58l1.244 1.209c-1.203 1.446-2.863 2.169-4.979 2.169-1.037 0-1.986-.16-2.847-.48a5.852 5.852 0 0 1-2.194-1.41c-.601-.62-1.068-1.39-1.4-2.31-.332-.919-.498-1.967-.498-3.145 0-1.198.171-2.262.514-3.192.342-.93.809-1.71 1.4-2.34a5.98 5.98 0 0 1 2.07-1.44 6.531 6.531 0 0 1 2.55-.496zm-4.294 6.012h7.81c0-.083.005-.16.016-.233.01-.072.015-.16.015-.263 0-.517-.098-1.007-.295-1.472a3.888 3.888 0 0 0-.794-1.209 3.9 3.9 0 0 0-1.167-.82 3.4 3.4 0 0 0-1.447-.31c-.975 0-1.856.345-2.645 1.037-.788.693-1.286 1.782-1.493 3.27zm19.692-6.012c.746 0 1.473.134 2.178.403a5.04 5.04 0 0 1 1.851 1.224c.53.547.955 1.245 1.276 2.092.322.847.482 1.849.482 3.006a8.676 8.676 0 0 1-.062 1.022H72.225c.042.992.213 1.823.514 2.495.3.671.679 1.214 1.135 1.627.457.413.965.707 1.525.883a5.59 5.59 0 0 0 1.68.263c1.473 0 2.697-.526 3.672-1.58l1.245 1.209c-1.203 1.446-2.863 2.169-4.979 2.169-1.037 0-1.986-.16-2.847-.48a5.852 5.852 0 0 1-2.194-1.41c-.601-.62-1.068-1.39-1.4-2.31-.332-.919-.498-1.967-.498-3.145 0-1.198.171-2.262.513-3.192.343-.93.81-1.71 1.4-2.34a5.98 5.98 0 0 1 2.07-1.44 6.531 6.531 0 0 1 2.552-.496zm-4.295 6.012h7.81c0-.083.006-.16.016-.233.01-.072.016-.16.016-.263a3.888 3.888 0 0 0-1.09-2.68 3.9 3.9 0 0 0-1.166-.822 3.4 3.4 0 0 0-1.447-.31c-.975 0-1.857.346-2.645 1.038-.788.693-1.286 1.782-1.494 3.27zm18.2 5.795c.373.372.56.795.56 1.27 0 .476-.182.889-.545 1.24a1.832 1.832 0 0 1-1.322.527c-.519 0-.954-.17-1.307-.512a1.678 1.678 0 0 1-.529-1.255c0-.495.187-.919.56-1.27.373-.351.799-.527 1.276-.527s.913.176 1.307.527zm4.386-11.466h7.562v13.48c0 1.074-.145 1.989-.436 2.743-.29.754-.69 1.368-1.198 1.844a4.974 4.974 0 0 1-1.789 1.069 7.98 7.98 0 0 1-2.21.418c-1.825 0-3.277-.661-4.356-1.983l1.463-1.922c.104.104.16.207.17.31.011.104.048.217.11.341.062.124.187.264.373.418.187.155.498.347.934.574.456.227.944.34 1.462.34.934 0 1.701-.314 2.303-.945.602-.63.902-1.657.902-3.083V59.274h-5.29v-1.828zm5.29-5.702c.312-.31.695-.465 1.152-.465.435 0 .814.155 1.136.465.321.31.482.692.482 1.146 0 .434-.16.806-.482 1.116-.322.31-.7.465-1.136.465-.436 0-.815-.155-1.136-.465a1.494 1.494 0 0 1-.482-1.116c0-.454.155-.836.466-1.146zm18.557 7.592l-1.307 1.952c-.104-.02-.155-.103-.155-.247 0-.145-.021-.248-.063-.31-.373-.476-.902-.915-1.587-1.317-.684-.403-1.462-.605-2.333-.605a4.56 4.56 0 0 0-1.136.14 3.688 3.688 0 0 0-.965.387 1.97 1.97 0 0 0-.669.635 1.598 1.598 0 0 0-.249.884c0 .248.062.47.187.666.124.196.321.387.591.573.27.186.648.372 1.136.558.487.186 1.125.392 1.914.62 1.784.475 3.044 1.038 3.78 1.689.737.65 1.105 1.451 1.105 2.401 0 .62-.135 1.209-.405 1.767a4.177 4.177 0 0 1-1.182 1.456c-.519.413-1.141.739-1.867.976-.726.238-1.546.357-2.458.357-2.386 0-4.388-.744-6.006-2.231l1.276-2.232c.062.062.104.16.124.295.021.134.094.294.218.48.125.186.353.398.685.635.332.238.84.501 1.524.79a3.96 3.96 0 0 0 1.058.31c.395.062.789.093 1.183.093.415 0 .835-.051 1.26-.155a4.38 4.38 0 0 0 1.152-.449c.342-.196.617-.434.824-.713.208-.279.311-.594.311-.945 0-.537-.249-.991-.747-1.363-.497-.372-1.41-.754-2.738-1.147a42.994 42.994 0 0 1-1.805-.558c-.746-.289-1.286-.542-1.618-.759a4.586 4.586 0 0 1-.902-.775c-.27-.3-.488-.64-.654-1.022a3.117 3.117 0 0 1-.249-1.255c0-.496.135-.971.405-1.426.27-.454.648-.857 1.136-1.209a6.216 6.216 0 0 1 1.742-.852 7.126 7.126 0 0 1 2.194-.325c2.178 0 3.942.744 5.29 2.231z"
                fill="#FFF"
                fillRule="evenodd"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="main__mdc-layout-grid___gQcgz links"
      >
        <div
          className="main__mdc-layout-grid__inner___1ZRyv"
        >
          <div
            className="main__mdc-layout-grid__cell___fCP2v main__mdc-layout-grid__cell--span-8-tablet___TJQhx main__mdc-layout-grid__cell--span-12-desktop___3LWuk main__mdc-layout-grid__cell--align-middle___2IP35"
          >
            <div
              className="contact"
              data-jsx={3457201880}
            >
              <p
                className="icon-separator"
                data-jsx={3457201880}
              >
                <svg
                  height="80"
                  viewBox="0 0 80 80"
                  width="80"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <g
                    fill="none"
                    fillRule="evenodd"
                    stroke="#FFF"
                    strokeWidth="2"
                    transform="translate(17 17)"
                  >
                    <ellipse
                      cx="36.892"
                      cy="10"
                      rx="10.108"
                      ry="10"
                    />
                    <path
                      d="M41.18 14.288c2.344-2.343 2.323-6.162-.045-8.53M35.124 13.75L3.79 44.25M28.048 15L10.613 28.5M32.597 1L0 20M44.978 15.75L26.785 47"
                      strokeLinecap="round"
                      strokeLinejoin="round"
                    />
                  </g>
                </svg>
              </p>
              <p
                data-jsx={3457201880}
              >
                I’m always happy 
                <a
                  data-jsx={3457201880}
                  href="mailto:hello@anotherplanet.io"
                >
                  to be involved
                </a>
                 into interesting projects.
              </p>
              <p
                data-jsx={3457201880}
              >
                <b
                  data-jsx={3457201880}
                >
                  Say hello:
                </b>
                 
                <span
                  data-jsx={3757623553}
                >
                  <span
                    data-jsx={3757623553}
                  >
                    <a
                      data-jsx={3757623553}
                      href="https://twitter.com/Autre_planete?ref=anotherplanet"
                      onClick={[Function]}
                      rel="noopener"
                      target="_blank"
                    >
                      twitter
                    </a>
                    , 
                  </span>
                  <span
                    data-jsx={3757623553}
                  >
                    <a
                      data-jsx={3757623553}
                      href="https://medium.com/@anotherplanet?ref=anotherplanet"
                      onClick={[Function]}
                      rel="noopener"
                      target="_blank"
                    >
                      medium
                    </a>
                    , 
                  </span>
                  <span
                    data-jsx={3757623553}
                  >
                    <a
                      data-jsx={3757623553}
                      href="https://github.com/thierryc?ref=anotherplanet"
                      onClick={[Function]}
                      rel="noopener"
                      target="_blank"
                    >
                      github
                    </a>
                    , 
                  </span>
                  <span
                    data-jsx={3757623553}
                  >
                    <a
                      data-jsx={3757623553}
                      href="https://fr.pinterest.com/apcx/?ref=anotherplanet"
                      onClick={[Function]}
                      rel="noopener"
                      target="_blank"
                    >
                      pinterest
                    </a>
                    , 
                  </span>
                  <span
                    data-jsx={3757623553}
                  >
                    <a
                      data-jsx={3757623553}
                      href="mailto:hello@anotherplanet.io"
                      onClick={[Function]}
                      rel="noopener"
                      target="_blank"
                    >
                      email
                    </a>
                    .
                  </span>
                </span>
              </p>
            </div>
          </div>
        </div>
      </div>
    </main>
  </div>
  <footer
    data-jsx={571573584}
  >
    <p
      data-jsx={571573584}
      style={
        Object {
          "textAlign": "center",
        }
      }
    >
      <span>
        <a
          className="twitter-follow-button"
          data-show-count="false"
          data-size="large"
          href="https://twitter.com/Autre_planete"
          onClick={[Function]}
        >
          Follow @
          Autre_planete
        </a>
        <script
          async={true}
          charSet="utf-8"
          src="//platform.twitter.com/widgets.js"
        />
      </span>
    </p>
    <p
      data-jsx={571573584}
    >
      © 1994 - 
      2017
       Thierry Charbonnel - All Rights Reserved
    </p>
  </footer>
</div>
`;
